pkg/agent/periodicstatusupdate.go

```diff
@@ -20,6 +22,12 @@ func (su *periodicStatusUpdate) Ready() bool {
 	if runStatusUpdateCheck() != nil {
 		return false
 	}
+	// Do not start until status will be running// get the status from the health check and jobs
```

Comment:
what happened here? 2 lines of comments get merged into 1 line accidentally?

Reply:
looks like it, will fix